src/components/Form/Form.tsx

```diff
@@ -1,62 +1,153 @@
-import CheckRadioConcept from "./CheckRadioConcept";
-import RangeConcept from "./RangeConcept";
-import { CHECK_AND_RADIO, RANGE } from "../../data";
-import { ICardData } from "../Cards";
+import { CHECK_AND_RADIO } from "../../data";
 import Alert from "../Alert";
-import React from "react";
+import React, { useState } from "react";
+import Box from "@mui/material/Box";
+import Grid from "@mui/material/Grid";
+import Slider from "@mui/material/Slider";
+import Input from "@mui/material/Input";
+import { Theme, useTheme } from "@mui/material/styles";
+import OutlinedInput from "@mui/material/OutlinedInput";
+import InputLabel from "@mui/material/InputLabel";
+import MenuItem from "@mui/material/MenuItem";
+import FormControl from "@mui/material/FormControl";
+import Select, { SelectChangeEvent } from "@mui/material/Select";
+import Chip from "@mui/material/Chip";
+import Button from "@mui/material/Button";
+import ClearIcon from "@mui/icons-material/Clear";
+import SendIcon from "@mui/icons-material/Send";
+import Stack from "@mui/material/Stack";
 
+export interface AI {
+  levelOfAI: string[];
+  whereAIIsUsed: string[];
+  TypeOfAI: string;
+  rateAIIntelligence: number;
+}
 interface IFormProps {
-  setFormData: (data: (prevData: ICardData[]) => ICardData[]) => void;
+  onSubmit: (Ai: AI) => void;
+}
+
+const ITEM_HEIGHT = 48;
+const ITEM_PADDING_TOP = 8;
+const MenuProps = {
+  PaperProps: {
+    style: {
+      maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+      width: 250,
+    },
+  },
+};
+
+function getStyles(name: string, option: readonly string[], theme: Theme) {
+  return {
+    fontWeight:
+      option.indexOf(name) === -1
+        ? theme.typography.fontWeightRegular
+        : theme.typography.fontWeightMedium,
+  };
```

Comment:
what this fucn should do?

Reply:
this function changes font for option that choosed. If choosed -> fontWeight =500, else -> fontWeight = 400

![image](https://github.com/proga-react-practice/xi-maxym-matskiv/assets/123649362/0353d16b-de4a-4811-a4ee-972e840c1b43)

Reply:
this a bit dirty hack with mui you could simply use

```
    sx={{
      "&.Mui-selected": {
        background: "red",
      },
    }}
```